### PR TITLE
Remove duplicate application.html.erb layout

### PR DESCRIPTION
**Pull request by alex-parkhamovich**
Why:
* App creates both application.html.erb and application.html.slim layouts

How:
* Remove typo to make generation working correct

### CODE REVIEW

File: lib/underlay/version.rb [hunk 1]
```diff
@@ -6,5 +6,5 @@ module Underlay
     .read("#{File.dirname(__FILE__)}/../../.ruby-version")
     .strip
     .freeze
-  VERSION = '1.45.0'
+  VERSION = '1.46.0'
```
Comment (vitaliiprodan) [inline lib/underlay/version.rb]: I'd say it's bug fix, so it could be `'1.45.1'`, but it's not a big problem now

Comment (alex-parkhamovich) [reply lib/underlay/version.rb]: @vitaliiprodan good point, fixed

**Comment by alex-parkhamovich** [comment]
@vitaliiprodan good point, thanks. I've removed that line from readme

**Comment by alvaro-chalar** [comment]
@alex-parkhamovich Instead of removing the line, I think it is better to mark it as done with an `X`

**Comment by alex-parkhamovich** [comment]
@alvaro-chalar sounds sensible, fixed

**Pull request merged**